com.unity.ml-agents/Runtime/Policies/HeuristicPolicy.cs

```diff
@@ -35,5 +41,88 @@ public float[] DecideAction()
         public void Dispose()
         {
         }
+
+        /// <summary>
+        /// Trivial implementation of the IList interface that does nothing.
+        /// This is only used for "writing" observations that we will discard.
+        /// </summary>
+        class NullList : IList<float>
```

Comment:
This is a bit "big" but seems liked a clean/fast way to ignore all the generated observations, without needing to allocate any memory buffers.

Reply:
It's the IList equivalent of `/dev/null`